packages/contracts/src/plugins/governance/admin/Admin.sol

```diff
@@ -71,4 +71,9 @@ contract Admin is IMembershipContract, PluginCloneable, Proposal {
         });
         _executeProposal(dao, proposalId, _actions, _allowFailureMap);
     }
+
+    /// @dev This empty reserved space is put in place to allow future versions to add new
+    /// variables without shifting down storage in the inheritance chain.
+    /// https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps
```

Comment:
```suggestion
    /// @notice This empty reserved space is put in place to allow future versions to add new variables without shifting down storage in the inheritance chain (see [OpenZepplins guide about storage gaps](https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps)).
```
This should be a `/// @notice` to be formatted correctly in our dev-portal. Please make it a one-line comment.